src/js/core/playRacing.js

```diff
@@ -0,0 +1,31 @@
+import Car from './car.js';
+import { RAMDOM_MIN, RAMDOM_MAX } from '../constants/constant.js';
+import { renderResult, renderWinners } from '../view/renderResult.js';
+
+export const makeCar = carNames => {
```

Comment:
자꾸만 변수명에 대한 코멘트를 드려서 죄송해요ㅠ 그만큼 중요하고 어렵기 때문입니다. 현업에서도 네이밍 하나 때문에 회의 열고 그럽니다ㅋㅋ...

단수/복수도 함수의 역할을 유추할 수 있는 중요한 단서 중 하나입니다!

```suggestion
export const makeCars = carNames => {
```

Reply:
반영했습니다!